packages/pancake-uikit/src/hooks/useTooltip/StyledTooltip.tsx

```diff
@@ -23,7 +23,7 @@ export const StyledTooltip = styled.div`
   line-height: 130%;
   border-radius: 16px;
   max-width: 320px;
-  z-index: 10;
+  z-index: 101;
```

Comment:
Set z-index to 101 because our modals have z-index 100, i.e. if we have tooltip in modal it would have appeared behind the modal.
_Teeechnically_ it could have been 100, cause `portal-root` div comes after react app's `root` and when we have same z-indices the one coming later in DOM tree would appear later... but I don't trust it 👀